### PR TITLE
Some fixes for R API documentation

### PR DESCRIPTION
**Pull request by smurching**
## What changes are proposed in this pull request?
 
* Enhances docs for `client` argument to tracking APIs, `mlflow_param()` and `mlflow_run()` APIs
* Pulls in a yet-to-be-released fix in Rd2md for rendering of R code examples (see https://github.com/quantsch/Rd2md/issues/1 for context)
* Eliminates some `seealso` sections from R docs
* Adds link to R homepage at top of R API docs.
* Also fixes R tutorial to account for renaming of arguments to `mlflow_run` in #1265
 
## How is this patch tested?
 
Running `make rdocs` and `make livehtml` from the `docs` directory & viewing generated docs.
 
### Is this a user-facing change? 

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.
 
(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
 
### What component(s) does this PR affect?
 
- [ ] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [ ] Examples 
- [x] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [ ] Models 
- [ ] Scoring 
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by mparkhe — COMMENTED** [review]
Left some comments in `R-api.rst`, though if I understand right this file is autogenerate and the source of the data is elsewhere in `*.R*` files. Does it make sense to maintain a single source of truth in git repo and remove `R-api.rst` file from the repo?

Overall the rendered docs look great! Thanks @smurching 

cc: @stbof

File: docs/source/R-api.rst [hunk 1]
```diff
@@ -109,28 +118,28 @@ experiment are also deleted.
...
-
-Details
--------
++-------------------------------+--------------------------------------+
+| Argument                      | Description                          |
++===============================+======================================+
+| ``experiment_id``             | ID of the associated experiment.     |
+|                               | This field is required.              |
++-------------------------------+--------------------------------------+
+| ``client``                    | (Optional) An ``mlflow_client``      |
```
[the remark this answers is not in the document]
Comment (smurching) [reply docs/source/R-api.rst]: ah that's a good idea - I think it's possible, let me give it a stab

**Review by tomasatdatabricks — APPROVED** [review]
lgtm

File: docs/source/R-api.rst [hunk 2]
```diff
@@ -481,11 +503,15 @@ Arguments
 +----------------+------------------------------------------------------------+
 | Argument       | Description                                                |
 +================+============================================================+
+| ``flavor``     | An MLflow flavor object.                                   |
```
[the remark this answers is not in the document]
Comment (smurching) [reply docs/source/R-api.rst]: Good point, tried to clarify the role of this API & the flavor arg (it's used internally but exposed so users can implement custom flavors)

File: docs/source/R-api.rst [hunk 3]
```diff
@@ -417,21 +438,22 @@ Arguments
...
-When ``client`` is not specified, these functions attempt to infer the
-current active client.
+``mlflow_list_run_infos``
+=========================
 
 List Run Infos
-==============
 
-List run infos.
+Returns a tibble whose columns contain run metadata (run ID, etc) for
```
[the remark this answers is not in the document]
Comment (smurching) [reply docs/source/R-api.rst]: Good catch, this is being fixed in parallel in #1310

**Pull request merged**